test/per_file_coverage.sh

```diff
@@ -52,8 +52,9 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/quic_listeners:85.0"
 "source/extensions/quic_listeners/quiche:84.8"
 "source/extensions/stat_sinks/statsd:85.2"
-"source/extensions/tracers:96.4"
+"source/extensions/tracers:96.3"
```

Comment:
skywalking tracer has higier coverage (>= 98.5) but the overall tracer coverage had decreased because remaining 1.5 ratio. So changed this value to 96.3.